session_items.py

```diff
@@ -1,73 +1,111 @@
 from flask import session
+import os
+import requests
+
+from Task import Task, Status
 
 _DEFAULT_ITEMS = [
     { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
     { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
 ]
 
+BOARD_ID = os.environ.get('BOARD_ID')
+TRELLO_KEY = os.environ.get("TRELLO_KEY")
+TRELLO_TOKEN = os.environ.get("TRELLO_TOKEN")
+
+TODO_LIST_ID = os.environ.get("TODO_LIST_ID")
+DONE_LIST_ID = os.environ.get("DONE_LIST_ID")
+
+trello_baseurl = "https://api.trello.com/"
+trello_apiversion = '1'
+trello_commonurl = trello_baseurl + trello_apiversion
+boardselector = "/board/"
+listselector = "/lists/"
+cardselector = "/cards/"
+
+default_query_params = {
+    'key': TRELLO_KEY,
+    'token': TRELLO_TOKEN,
+}
+
+statusToID = {
+    Status.ToDo: TODO_LIST_ID,
+    Status.Done: DONE_LIST_ID
+}
+IDToStatus = {
+    TODO_LIST_ID: Status.ToDo,
+    DONE_LIST_ID: Status.Done
+}
 
 def get_items():
```

Comment:
These methods could be `@staticmethod`s on a TrelloApi class.

Reply:
When i try to set that up I get `AttributeError: module 'TrelloApi' has no attribute 'get_items'` at runtime, and I can't see why, I haven't renamed the function, I've marked it as static
I'm not quite sure the benefit of it either, if every method is static. Is it purely to be consistently object-oriented?

Reply:
Essentially you're right, it's really only for namespacing the method under a class (which some might argue is better than using the file as a namespace).

Reply:
You'd have to send me a code snippet for me to help you with that error though.